components/pedigree/resources/src/main/resources/pedigree/view/cancersLegend.js

```diff
@@ -51,7 +51,7 @@ define([
                                    "Myeloma" : "Myeloma",
                                    "Leukemia" : "Leukemia"
                                  };
-            $super('Cancers', true);
```

Comment:
well, it is kind of related as this enables better "rejection" message. PT-2825 is not a bug, just the message is a bit unclear,and in case of group node there is no message at all. All of these changes are related to improving the messages. I guess a better "fix" is to re-word PT-2825 itself